.gitignore

```
@@ -2,5 +2,5 @@
 composer.phar
 composer.lock
 .DS_Store
-clover.xml
+.coverage.xml
```

```suggestion
coverage.xml
```

Extra "." on the beginning.